pkgs/applications/misc/geoipupdate/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "geoipupdate";
+  version = "4.1.5";
+
+  src = fetchFromGitHub {
+    owner = "maxmind";
+    repo = "geoipupdate";
+    rev = "v${version}";
+    sha256 = "1k0bmsqgw35sdmaafinlr4qd5910fi598i8irxrz11394d3c8giv";
+  };
+
+  modSha256 = "0jlg2p5qzfrw64ay5fphigf36yf67rjyw05vj5xhkqa630ph5ydr";
+
+  meta = with stdenv.lib; {
+    description = "Automatic GeoIP database updater";
+    license = with licenses; [ asl20 ];
```

Comment:
Sorry, one more :-/

`    homepage = "https://github.com/maxmind/geoipupdate"`;

Reply:
I thought about that but I wasn't sure if this is really the homepage :/

Reply:
It's just some way to link to the correct repo, as there may be other repos with the same name. github is commonly used as a landing page